packages/app-shortcut/src/definitions.ts

```diff
@@ -56,6 +56,7 @@ export interface Shortcut {
   description?: string;
   id: string;
   title: string;
+  iosIcon?: number;
```

Comment:
Is this mandatory on iOS or can we skip it in the first version?

Reply:
The description and the icon are needed together. To eliminate the icon, we should remove the description (subtitle in the code) as well.

Reply:
Okay, in this case remove the description or support it only on Android (this is up to you).

Reply:
I will support description only on android in this case